impl/client/src/main/javascript/web/dojo/pentaho/common/TemplatePicker.js

```diff
@@ -80,6 +88,17 @@ define(["dojo/_base/declare", "dijit/_WidgetBase", "dijit/_Templated", "dojo/on"
               on(this.templateName3, 'click',lang.hitch( this, 'imgClick'));
               on(this.templateName4, 'click',lang.hitch( this, 'imgClick'));
               on(this.templateName5, 'click',lang.hitch( this, 'imgClick'));
+              this.own(
```

Comment:
I'm glad you are using `this.own(...)`. However, please also pass to it the results of above `on(.)` calls.